BankSDK/GiniBankSDKExample/GiniBankSDKExample/SelectAPIViewController.swift

```diff
@@ -10,44 +10,256 @@ import GiniCaptureSDK
     }
     
+    @IBAction func photoPaymentButtonTapped(_ sender: Any) {
+        startSDK(entryPoint: .button)
+    }
+    
+    @objc func ibanCameraIconTapped(_ sender: Any) {
+        startSDK(entryPoint: .field)
+    }
+    
+    // MARK: - Notifications
+    
+    private func addObservers() {
+        NotificationCenter.default.addObserver(self,
+                                               selector: #selector(keyboardWillShow(notification:)),
+                                               name: UIResponder.keyboardWillShowNotification,
+                                               object: nil)
+        
+        NotificationCenter.default.addObserver(self,
+                                               selector: #selector(keyboardWillHide(notification:)),
+                                               name: UIResponder.keyboardWillHideNotification,
+                                               object: nil)
+    }
+    
+    private func removeObservers() {
+        NotificationCenter.default.removeObserver(self, name: UIResponder.keyboardWillShowNotification, object: nil)
+        NotificationCenter.default.removeObserver(self, name: UIResponder.keyboardWillHideNotification, object: nil)
+    }
+    
+    // MARK: - Keyboard handling
+    
+    @objc func keyboardWillShow(notification: Notification) {
+        guard let keyboardFrame = (notification.userInfo?[UIResponder.keyboardFrameEndUserInfoKey] as? CGRect) else { return }
+        
+        if let focusedTextField = focusedFormField, view.frame.origin.y == 0 {
+            
+            let textFieldFrameInParent = view.convert(focusedTextField.frame, from: focusedTextField.superview)
+            let remainingHeight = view.frame.height - textFieldFrameInParent.height - textFieldFrameInParent.origin.y
+            if keyboardFrame.height > remainingHeight {
+                view.frame.origin.y -= keyboardFrame.height - remainingHeight
+                view.layoutIfNeeded()
+            }
+        }
+    }
+    
+    @objc func keyboardWillHide(notification: Notification) {
+        if view.frame.origin.y != 0 {
+            view.frame.origin.y = 0
+            view.layoutIfNeeded()
+        }
+    }
+    
+    private func initializeHideKeyboard() {
+        //Declare a Tap Gesture Recognizer which will trigger our dismissMyKeyboard() function
+        let tap: UITapGestureRecognizer = UITapGestureRecognizer(
+            target: self,
+            action: #selector(dismissKeyboard))
+        
+        //Add this tap gesture recognizer to the parent view
+        view.addGestureRecognizer(tap)
+    }
+    
+    @objc func dismissKeyboard() {
+        //endEditing causes the view (or one of its embedded text fields) to resign the first responder status.
+        //In short- Dismiss the active keyboard.
+        view.endEditing(true)
+        focusedFormField = nil
+    }
+    
+    // MARK: - Utils
+    
+    private func giniCaptureColor(_ name: String) -> UIColor {
```

Comment:
[Clarification]: Shall we keep `giniCaptureColor` here or move into Color utils?